Source/automap.h

```diff
@@ -78,22 +80,44 @@ enum class AmLineLength : uint8_t {
 	OctupleTile = 64,
 };
 
+enum class AutomapType : uint8_t {
+	Opaque,
+	Transparent,
+	Minimap,
+	LAST = Minimap
+};
+
+extern AutomapType CurrentAutomapType;

```

Comment:
You need something like this for MSVC to be able to acces the global during tests.
```suggestion
extern DVL_API_FOR_TEST AutomapType CurrentAutomapType;
```